src/main/java/net/pms/formats/Format.java

```diff
@@ -422,7 +422,7 @@ public static boolean isSupportedMimetype(String mimetype) {
 		if ("audio/L24".equalsIgnoreCase(mimetype)
 				|| "video/vnd.dlna.mpeg-tts".equalsIgnoreCase(mimetype)
 				|| "video/mpeg2".equalsIgnoreCase(mimetype)
-				|| "audio/vnd.dlna.adts".equals(mimetype)
+				|| "audio/vnd.dlna.adts".equalsIgnoreCase(mimetype)
```

Comment:
Defining `mimetype` as lower or upper case should "clean" the code as all `equalsIgnoreCase` could be replaced by `equals`